<ui version="4.0" >
 <class>searchLayout1Class</class>
 <widget class="QWidget" name="searchLayout1Class" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>312</width>
    <height>407</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>searchLayout1</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="spacing" >
    <number>4</number>
   </property>
   <property name="leftMargin" >
    <number>4</number>
   </property>
   <property name="topMargin" >
    <number>4</number>
   </property>
   <property name="rightMargin" >
    <number>4</number>
   </property>
   <property name="bottomMargin" >
    <number>4</number>
   </property>
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string/>
       </property>
       <property name="pixmap" >
        <pixmap resource="../qvortaro.qrc" >:/find</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="EoLineEdit" name="searchword" />
     </item>
     <item>
      <widget class="QPushButton" name="pbClear" >
       <property name="maximumSize" >
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="focusPolicy" >
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="icon" >
        <iconset resource="../qvortaro.qrc" >:/clear</iconset>
       </property>
       <property name="iconSize" >
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="flat" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="lexikonFrame" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
       <horstretch>9</horstretch>
       <verstretch>9</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape" >
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" >
      <property name="spacing" >
       <number>4</number>
      </property>
      <property name="leftMargin" >
       <number>0</number>
      </property>
      <property name="topMargin" >
       <number>0</number>
      </property>
      <property name="rightMargin" >
       <number>0</number>
      </property>
      <property name="bottomMargin" >
       <number>0</number>
      </property>
      <item>
       <widget class="EoListBrowser" native="1" name="lexikon1" />
      </item>
      <item>
       <widget class="EoListBrowser" native="1" name="lexikon2" />
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <customwidgets>
  <customwidget>
   <class>EoLineEdit</class>
   <extends>QLineEdit</extends>
   <header>eolineedit.h</header>
  </customwidget>
  <customwidget>
   <class>EoListBrowser</class>
   <extends>QWidget</extends>
   <header>eolistbrowser.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../qvortaro.qrc" />
 </resources>
 <connections/>
</ui>
